CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Changes by Version
 
 #### Backend Changes
 
+- The traces related metrics on collector now have a new tag `sampler_type` ([#1576](https://github.com/jaegertracing/jaeger/pull/1576))
+
+  this might break some existing metrics dashboard (if so, users need to update query to aggregate over this new tag).
+
```

Comment:
should this be under Breaking changes then?  

cc @pavolloffay @objectiser

Reply:
hmmm. Like what @black-adder mentioned in that PR, it would depends on how user configured their dashboard at first place.

Reply:
Updated